vowpalwabbit/core/src/reductions/plt.cc

```diff
@@ -139,6 +132,18 @@ void learn(plt& p, learner& base, VW::example& ec)
     }
   }
   else { p.negative_nodes.insert(0); }
+}
+
+void learn(plt& p, learner& base, VW::example& ec)
+{
+  auto multilabels = std::move(ec.l.multilabels);
+  VW::polyprediction pred = std::move(ec.pred);
+
+  double t = p.all->sd->t;
+  double weighted_holdout_examples = p.all->sd->weighted_holdout_examples;
+  p.all->sd->weighted_holdout_examples = 0;
```

Comment:
Why reset this to zero?

Reply:
Good question. This is there from the initial implementation, it is not a change I introduced in this PR, but I don't remember why it's here. Does it somehow impact the update of the base classifier? That probably could be a reason for "resetting" this variable. If not, this can probably be removed.

Reply:
Ok, it seems that it doesn't impact the training so I think I will remove these.

Reply:
Sounds good!